### PR TITLE
Add Graphics Workshop Event

### PR DESCRIPTION
**Pull request by nivasini**

### CODE REVIEW

File: content/W2016/W16-event-graphics-workshop.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Title: Graphics Workshop
+Date: 2016-03-01
+Category: Events
+Tags: workshop, graphics
+Slug: graphics-workshop-w16
+Author: Aurora Huang
```
Comment (fboxwala) [inline content/W2016/W16-event-graphics-workshop.md:6]: You should probs change this to be you

File: content/W2016/W16-event-graphics-workshop.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Summary: Express your creativity by making art on 3D animation software!
+
+Come out to WiCS graphics workshop! Express your creativity and have fun with your friends by making art on an animation software! No experience is required, a female instructor will give a Blender tutorial that walks you through step-by-step to create a simple 3D animation. Female mentors will also be present to provide guidance.
```
Comment (fboxwala) [inline content/W2016/W16-event-graphics-workshop.md]: Maybe make it "that walks you through the steps of creating a simple 3D animation.

File: content/W2016/W16-event-graphics-workshop.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+For this event you will need to bring your own computer and also have the latest Blender downloaded and installed: [https://www.blender.org/download/] (https://www.blender.org/download/)
+
+
+
+## Event Details ##
+
++ **Who:** Women and Non-binary individuals
```
Comment (ehashman) [inline content/W2016/W16-event-graphics-workshop.md]: This is inconsistent with how we normally advertise women-only events. Can you copy our usual description?

File: content/W2016/W16-event-graphics-workshop.md (hunk 2 above)
Comment (ehashman) [inline content/W2016/W16-event-graphics-workshop.md]: The "female instructor" and "female mentors" bits make me a bit uncomfortable. Can we change this so we use the instructor's name and don't make assumptions about our mentors' genders?

File: content/W2016/W16-event-graphics-workshop.md (hunk 1 above)
Comment (nivasini) [inline content/W2016/W16-event-graphics-workshop.md:6]: Aurora was the one who came up with the content. I just copy-pasted what she sent me

**Pull request merged**